### PR TITLE
Bug 972228 - Verify default file content in non-distribution test

### PR DESCRIPTION
**Pull request by rickychien**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cd0252875e2d)

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e3344aa085ec)

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=49df75c0f690)

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c65730edca2b)

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=95027ffb1d62)

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fb07b44e68e6)

File: apps/callscreen/test/build/integration/callscreen_test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+var helper = require('helper');
+
+suite('Callscreen tests', function() {
+  suiteSetup(helper.cleanupWorkspace);
+  teardown(helper.cleanupWorkspace);
+
+  // Do not check callscreen
+  // We're going to uncomment this once bug 1093267 is resolved
+  /*
+  test('APP=callscreen make', function(done) {
```
Comment (cctuan) [inline apps/callscreen/test/build/integration/callscreen_test.js]: just use test.skip.

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=572575ca9ab1)

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0c596dedf6d4)

**Pull request merged**